src/components/entity/ha-state-label-badge.ts

```diff
@@ -110,7 +110,9 @@ export class HaStateLabelBadge extends LitElement {
         return null;
       case "sensor":
       default:
-        return state.state === UNKNOWN
+        return state?.attributes.device_class === "moon__phase"
```

Comment:
state will always be defined, we return an error badge if it isnt.
```suggestion
        return state.attributes.device_class === "moon__phase"
```